hw/ip/spi_device/data/spi_device_testplan.hjson

```diff
@@ -14,154 +14,13 @@
     {
       name: smoke
       desc: '''
-            Use default SRAM fifo setting. Seq:
-            - Write a word data to TX memory and update wptr
-            - Send a word SPI transfer
-            - Read a word data from RX memory and update rptr
-            - Compare the data and check no pending data in SRAM FIFO
-            - Repeat above steps'''
+            Configures Flash/Passtrhough modes as well as TPM and drive reads/writes
+            through SPI alternating between TPM and SPI commands.
+            Sequences spi_device_flash_all_vseq and spi_device_tpm_all_vseq are
+            launched in parallel'''
       stage: V1
       tests: ["spi_device_smoke"]
```

Comment:
Hasn't the `spi_device_smoke` test been deleted as well? I guess it's `spi_device_flash_and_tpm` that now becomes the smoketest effectively? It might be worth moving this description to that test now.